src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -73,6 +73,7 @@ def register_submission(
                 "informatieobjecttype": "informatieobjecttype_submission_report",
                 "organisatie_rsin": "organisatie_rsin",
                 "vertrouwelijkheidaanduiding": "vertrouwelijkheidaanduiding",
+                "author": "author",
```

Comment:
why not map this to `auteur` directly so the right key ends up in the API calls?